kamailio/default.cfg

```diff
@@ -670,6 +674,10 @@ route[INTERNAL_TO_EXTERNAL_RELAY]
 
     remove_hf_re("^X-.*");
 
+    #!ifdef SIP_TRACE_ROLE
+    route(SEND_SIP_TRACE);
+    #!endif
+
```

Comment:
@sergey-safarov can this block be moved to route[RELAY] and delete the other identical block below ?